lua/cmake/utils.lua

```diff
@@ -89,7 +90,17 @@ function utils.get_current_target(parameters)
     return nil, nil
   end
 
-  local target_dir = vim.fn.fnamemodify(target, ':h')
+  local target_dir
+  local run_dir = parameters['runDir']
+  if run_dir == nil then
+    target_dir = vim.fn.fnamemodify(target, ':h')
+  else
+    local target_dir = Path:new(run_dir)
+    if target_dir:is_absolute() ~= 1 then
```

Comment:
I'm pretty sure that Path returns boolen. In lua 1 not equal to `true`.

Reply:
Ok. Thank you! I am still pretty new to lua and learning it as I go.